src/components/molecules/Tarif/index.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import { Tarif as TarifProps } from './types';
+import styles from './Tarif.css';
+import Typography from '@/components/atoms/Typography';
+import { useState } from 'react';
+
+export const links = () => {
+  return [{ rel: 'stylesheet', href: styles }];
+};
+
+type Props = TarifProps & React.HTMLAttributes<HTMLElement>;
+
+const Tarif = ({ name, dayPrice, minimumPrice, example }: Props) => {
+  const [isOpen, setOpen] = useState(false);
+  return (
+    <div className="tarif__container">
```

Comment:
Die `__container` hoeft hier eigenlijk niet aan. `.tarif` lijkt mij in principe voldoende